app/src/pages/inside/projectSettingsPageContainer/content/notifications/modals/addEditNotificationModal/addEditNotificationModal.jsx

```diff
@@ -189,6 +190,8 @@ const AddEditNotificationModal = ({
   const { trackEvent } = useTracking();
   const dispatch = useDispatch();
   const [isEditorShown, setShowEditor] = React.useState(data.notification.attributes.length > 0);
+  const isExistingLaunchNames = useSelector(isExistingLaunchNamesSelector);
```

Comment:
Can we select it inside of `LaunchNamesContainer` directly?
As I see these data doesn't used anywhere else in current component,